tests/e2e/support.py

```diff
@@ -1,17 +1,36 @@
 import os
 import random
-import string
+import string, yaml
 import subprocess
 from kubernetes import client, config
-import kubernetes.client
-from codeflare_sdk.utils.kube_api_helpers import _kube_api_error_handling
+from kubernetes.client.exceptions import ApiException
+from minio import Minio
+
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
 
 
 def get_ray_image():
     default_ray_image = "quay.io/rhoai/ray:2.23.0-py39-cu121"
     return os.getenv("RAY_IMAGE", default_ray_image)
 
 
+def get_disconnected_setup_env_variables():
```

Comment:
`get_setup_env_variables()` as it's not always executed in a disconnected environment?